### DISCUSSION TITLE
Why does education degree is a bit higher than the rest of education info in classic theme and how to fix this?

### DISCUSSION THREAD

**Discussion by Prometheus3375** [rendercv/rendercv#252] [Q&A] [answered]
For example:
![image](https://github.com/user-attachments/assets/b2b2b293-49ee-4751-930d-881504913158)

This is less noticeable in example CV:
![image](https://github.com/user-attachments/assets/9739f40e-01d2-4b0d-aeb2-9a95f65ab5ed)

Is there a way to fix this? Why it happens in the first place? I didn't find anything in .tex that can contribute to this.

**Comment by sinaatalay** [comment]
This happens when the `design.margins.entry_area.education_degree_width` field is too small. Could you please open an issue for this?

**Reply by Prometheus3375** [reply]
I am not sure this causes the issue. In OP width is 0.5cm. Here are example with 1cm (default) and 2cm respectively, the degree is still noticeably higher than the rest of the text.

![image](https://github.com/user-attachments/assets/25b240ec-0fc6-43b4-b093-a6b49a2cc87d)
![image](https://github.com/user-attachments/assets/78691831-4e6e-42d0-96d4-875e649eb26e)

**Reply by sinaatalay** [reply, marked as answer]
This issue should have been fixed with v2.

**Reply by Prometheus3375** [reply]
The issue is back in v2.4.
<img width="653" height="65" alt="image" src="https://github.com/user-attachments/assets/13f499c4-da4d-4148-aa15-ebfc6a0d40a9" />
Here is the same paragraph in v2.3
<img width="656" height="69" alt="image" src="https://github.com/user-attachments/assets/29dd4794-38ec-4423-b898-e9aea4b6a86f" />

**Reply by sinaatalay** [reply]
Very interesting, I cannot reproduce this somehow. Can you send the Typst source code of this?

**Reply by sinaatalay** [reply]
The issue is about the external link icon. We need to make it smaller.

**Reply by Prometheus3375** [reply]
Probably bigger? In v2.3 it is bigger and degree is on the same line as everything else.

**Reply by sinaatalay** [reply]
Font Awesome got updated from 6 to 7, that's why there is a difference. I will look into this before v2.6.

**Reply by Prometheus3375** [reply]
Just reminding you that the issue is still in v2.6.

**Reply by sinaatalay** [reply]
I forgot about it. It would be great if you could open an issue for this.

**Reply by Prometheus3375** [reply]
#603
